os_net_config/sriov_config.py

```diff
@@ -619,11 +622,25 @@ def get_vdpa_vhost_devices():
     return loads(stdout)['dev']
 
 
-def configure_vdpa_vhost_device(pci):
+def _configure_vdpa_queues(pci, vdpa_queues):
+    if not strutils.is_int_like(vdpa_queues):
+        msg = 'vdpa_queues config value must be an integer'
```

Comment:
please move the parameter validation to objects.py as mentioned in the above comments

Reply:
done